specification/components/schemas/parameters/Priority.yaml

```diff
@@ -0,0 +1,31 @@
+title: Details the priority that was specified in a search criteria
```

Comment:
I guess this parameter could be used elsewhere so should probably not reference "search criteria"